src/neubot/dash_impl.hpp

```diff
@@ -87,12 +87,12 @@ static inline void loop_request(Var<Transport> transport, int speed_kbit,
 
                     if (time_elapsed < 0) {
                         logger -> warn("Time elapsed can't be negative");
-                        cb(error, nullptr);
+                        cb(ValueError(), nullptr);
```

Comment:
I think it would be more appropriate to define a new error in `mlabns.hpp` header that indicates what is happening here and then use that error.